### ISSUE TITLE
Use of uninitialized value $v in concatenation (.) or string at

### ISSUE THREAD

**Issue by sirlattwich** [jalavoy/slack-kbbot#2]
![2017-03-05 20_57_53-sirlattwich sirlattwich-pi_](https://cloud.githubusercontent.com/assets/26206581/23590774/774340e8-01e6-11e7-8478-11e9e0b3978a.png)

hey there o/ 
loved your script, but it stopped working few days ago for me. i get the error displayed in the screenshot. i am unsure if this is because some perl stuff was updated and the script is using outdated variable declarations? unfortunately i am not experienced with perl at all. please let me know if you need further information from me.

**Comment by jalavoy** [comment]
I'm unable to recreate this myself. The line that it's erroring out on means that it didn't get a valid response from the zkillboard API.

If this is still happening can you run it with the debug flag and send me the output?

Something like:
`/bin/perl /home/sirlattwhich/EVE/slack-kbbot-master/kbbot.pl -d`

**Comment by jalavoy** [comment]
Also if you could send me the kb_type and target_id from the conf that would also be helpful.

**Comment by sirlattwich** [comment]
![2017-03-06 23_24_42-sirlattwich-pi - kitty](https://cloud.githubusercontent.com/assets/26206581/23632958/225583b4-02c4-11e7-8cbe-6beba9717ac4.png)

```
# the ID of the corporation or alliance you're running the script against. you can get this from visiting your corp or alliances kill feed on zkillboard. It'll be the number in the URL, This is Goons for example: https://zkillboard.com/alliance/824518128/
target_id=99005392
# the type of item target_id is, valid options are corporation or alliance
kb_type=alliance
# you would get this from creating a new incoming webhook in slack. Go to Configure Integrations > Incoming WebHooks > Setup a hook and grab the Webhook URL
slack_hook_url=https://hooks.slack.com/services/******************************
channel=#killmails
username=Germans-zKBbot
emoji=:smirk:
```

here is the output with debug flag and the conf as requested

**Comment by jalavoy** [comment]
It looks like they disabled the method I was using to pull new kills in favor of this RedisQ thing.

I don't play the game anymore and don't have a lot of time to rework it to use the new method. However, when I was still playing I actually switched off of my copy and started using this one:
`https://github.com/dimurgos/Slack-Eve-Killmail`

I just confirmed it's still working. I'd recommend using his instead. 

Apologies for not being able to fix this. I'll update the README to say as much.
